contrib/babelfishpg_tsql/sql/import_export_compatibility.sql

```diff
@@ -59,9 +59,15 @@ CREATE OR REPLACE FUNCTION is_member(IN a VARCHAR) RETURNS INT
 AS 'babelfishpg_tsql', 'is_member' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION is_member(IN VARCHAR) TO PUBLIC;
 
-CREATE OR REPLACE FUNCTION schema_id(IN schema_name VARCHAR) RETURNS INT
-AS 'babelfishpg_tsql', 'schema_id' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
-GRANT EXECUTE ON FUNCTION schema_id(IN VARCHAR) TO PUBLIC;
+-- Two declarations of schema_id based on number of parameters.
+--However, both call the same C function
+CREATE OR REPLACE FUNCTION schema_id()
```

Comment:
Can we have a single definition by having default value set to empty string in the second def?

Reply:
What we want is when no argument is passed the function should return the ID of the default schema of the caller,  while in case of empty string the function will return NULL. If we set a default value to the argument, then we will not be able to differentiate btw no argument and empty string.

Reply:
I think we can define a single function with default input as NULL, but in that case `schema_id(NULL)`  will return `default` value instead of `NULL`

Reply:
I think we need appropriate comments in the code as well